gem.md

```diff
@@ -97,6 +97,15 @@ Gem Semantics
       requires VALUE >=Rat 0
        andBool BALANCE_SRC >=Rat VALUE
```

Comment:
In dai.md above, there is also the check that the two accounts must be different, even though we are matching on two entries in the balances map. For consistency, we should probably do the same here: `ACCTSRC =/=K ACCTDST`, right?

Reply:
Fixed.